packages/camera/camera/android/src/main/java/io/flutter/plugins/camera/features/regionboundaries/RegionBoundariesFeature.java

```diff
@@ -0,0 +1,94 @@
+// Copyright 2013 The Flutter Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+package io.flutter.plugins.camera.features.regionboundaries;
+
+import android.annotation.TargetApi;
+import android.hardware.camera2.CaptureRequest;
+import android.os.Build;
+import android.util.Size;
+import io.flutter.plugins.camera.CameraProperties;
+import io.flutter.plugins.camera.features.CameraFeature;
+import java.util.Arrays;
+
+/**
+ * Holds the current region boundaries. When this is created, you must provide a
+ * CaptureRequestBuilder for which we can read the distortion correction settings from.
+ */
+public class RegionBoundariesFeature extends CameraFeature<Size> {
+  private final CameraRegions cameraRegions;
+  private Size currentSetting;
+
+  public RegionBoundariesFeature(
+      CameraProperties cameraProperties, CaptureRequest.Builder requestBuilder) {
+    super(cameraProperties);
+
+    // No distortion correction support
+    if (android.os.Build.VERSION.SDK_INT < Build.VERSION_CODES.P
```

Comment:
Why not just make this check internal to supportsDistortionCorrection? Assuming false is always going to be the fallback, presumably.

Reply:
This is to silence the linter, since the `else` construct contains code that is only available on API level `P` and higher.

Reply:
Ah, I missed that this was a `<` check; usually it's the opposite. I think it would be clearer to flip the bodies and use
```
if (android.os.Build.VERSION.SDK_INT >= Build.VERSION_CODES.P && supportsDistortionCorrection()) {
  // the distortion correction code
} else {
  // the fallback code
}